pages/index.js

```diff
@@ -434,511 +448,459 @@ export default class IndexPage extends React.Component {
+                      </div>
+                    </div>
+                  </a>
+                </div>
+                <div css={STYLES_SLATE_CARD}>
+                  <a
+                    css={STYLES_SLATE_CARD_PARAGRAPH}
+                    href="https://slate.host/tcosta/montreal-underground"
+                    target="_blank"
+                  >
+                    <div css={STYLES_SLATE_CARD_TEXT}>
+                      <div css={STYLES_SLATE_CARD_TITLE}>Montreal underground</div>
+                      <div css={STYLES_SLATE_CARD_EXPLAINER}>
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>@tcosta</div>
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>-&gt;</div>
+                      </div>
+                    </div>
+                  </a>
+                </div>
+              </div>
+              <div css={STYLES_SLATE_CARD_GROUP}>
+                <div css={STYLES_SLATE_CARD}>
+                  <a css={STYLES_SLATE_CARD_PARAGRAPH} href="https://slate.host/slate/monet" target="_blank">
+                    <div css={STYLES_SLATE_CARD_TEXT}>
+                      <div css={STYLES_SLATE_CARD_TITLE}>Monet</div>
+                      <div css={STYLES_SLATE_CARD_EXPLAINER}>
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>@slate</div>
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>-&gt;</div>
+                      </div>
+                    </div>
+                  </a>
+                </div>
+                <div css={STYLES_SLATE_CARD}>
+                  <a css={STYLES_SLATE_CARD_PARAGRAPH} href="https://slate.host/haris/books" target="_blank">
+                    <div css={STYLES_SLATE_CARD_TEXT}>
+                      <div css={STYLES_SLATE_CARD_TITLE}>Books</div>
+                      <div css={STYLES_SLATE_CARD_EXPLAINER}>
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>@haris</div>
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>-&gt;</div>
+                      </div>
+                    </div>
+                  </a>
                 </div>
-              </a>
-            </div>
-            <br />
-            <br /> <div id="example-slates" css={STYLES_SLATE_CARD_GROUP}></div>
+                <div css={STYLES_SLATE_CARD}>
+                  <a css={STYLES_SLATE_CARD_PARAGRAPH} href="https://slate.host/slate/papers" target="_blank">
+                    <div css={STYLES_SLATE_CARD_TEXT}>
+                      <div css={STYLES_SLATE_CARD_TITLE}>Papers</div>
+                      <div css={STYLES_SLATE_CARD_EXPLAINER}>
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>@slate</div>
+                        <div css={STYLES_SLATE_CARD_PARAGRAPH}>-&gt;</div>
+                      </div>
+                    </div>
+                  </a>
+                </div>
+              </div>
+            </section>
           </div>
+          <WebsitePrototypeFooter />
```

Comment:
Why is <WebsitePrototypeFooter /> inside of this div?

Reply:
this is something Martina suggested to fix the weird grey background thing. 
Did I not implement this correctly?